contracts/SportMarkets/Rundown/GamesPlayerProps.sol

```diff
@@ -313,7 +313,6 @@ contract GamesPlayerProps is Initializable, ProxyOwned, ProxyPausable {
             _player.gameId,
             _append(_player), // gameLabel
             consumer.getGameCreatedById(_player.gameId).startTime, //maturity
-            0, //initialMint
```

Comment:
I would leave this to not have to release all these contracts

Reply:
done.